src/sentry/api/serializers/models/group.py

```diff
@@ -148,10 +148,23 @@ def get_attrs(self, item_list, user):
             environment = self.environment_func()
         except Environment.DoesNotExist:
             user_counts = {}
+            times_seen = {}
         else:
-            environment_id = environment and environment.id
+            project_id = item_list[0].project_id
+            item_id_list = [g.id for g in item_list]
```

Comment:
nit: can this just be `item_ids`, which lines up a little closer with the parameter name being `group_ids` (since we're using `item_` here since it's already in the context of group, otherwise I'd suggest `group_ids`)

Reply:
I don't mind either way (I flip flopped back and forth myself, settling with `item_` here for consistency with the other variable references) but am open to changing it

Reply:
Realized I misinterpreted this comment earlier, it's fixed here anyway 7ea6bc5578ffca5fd7708c75587c8e3e1dd8166a